code/style.css

```diff
@@ -1,58 +1,377 @@
-body {
+* {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  box-sizing: border-box;
+}
+
+/*** GENERAL ***/
+
+body {
+  font-family: 'Courier New', Courier, monospace;
+  background-color: #000;
 }
 
 p {
+  width: 100%;
   margin: 1px;
   font-weight: 300;
   background-color: #fff;
 }
 
-.cards {
-  margin: 50px auto;
+.visible {
+  display: flex;
+}
+
+.invisible {
+  display: none;
+}
+
+/*** HEADER WINDOW ***/
+
+.header-wrapper {
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 100px;
+}
+
+.header-text {
+  font-size: 6.5em;
+  color: crimson;
+  margin-bottom: -50px;
+}
+
+/*** DRIPPING ANIMATION ***/
+
+.drop {
+  position: relative;
+  width: 20px;
+  height: 20px;
+  top: 5px;
+  margin: 0 0 0 360px;
+  background: crimson;
+  -moz-border-radius: 20px;
+  -webkit-border-radius: 20px;
+  border-radius: 20px;
+  -moz-animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+  -webkit-animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+  animation: drip 3s cubic-bezier(1, 0, 0.91, 0.19) infinite;
+}
+
+.drop:before {
+  content: '';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 30px solid crimson;
+  top: -22px;
+}
+
+/*** HEADER BUTTON ***/
+
+.button-start {
+  color: #fff;
+  -moz-animation: pulse 5s infinite;
+  -webkit-animation: pulse 5s infinite;
+  animation: pulse 5s infinite;
+  cursor: pointer;
+  transition: transform 0.2s;
+}
+
+.button-start:hover {
+  transform: scale(1.2);
+  animation: none;
+}
+
+/*** GAMEBOARD WINDOW ***/
+
+.game-wrapper {
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  margin: 0 auto;
+  text-align: center;
+}
+
+/*** CARDS ***/
+
+.cards-container {
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
+  flex-direction: column;
+  max-width: 1100px;
+  margin: auto;
+  padding: 16px;
 }
 
-.killer-deck {
+.info-header {
+  color: #fff;
+  margin-top: 50px;
+  margin: auto;
+  padding: 20px;
+}
+
+.info {
+  color: #fff;
+  background: none;
+}
+
+.decks {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   width: 250px;
   height: 400px;
-  background: #000;
   text-align: center;
   padding-top: 30px;
+  padding-bottom: 20px;
   border-radius: 16px;
+  cursor: pointer;
+  background: #000;
+  margin: 25px 0;
```

Comment:
I like the way to collect similar styling of the decks into one class. Brilliant!